src/library/utils/apiUtils.js

```diff
@@ -2,21 +2,30 @@ import axios from 'axios';
 import qs from 'qs';
 //import { API_ROOT } from '../../env';
 import { HOST } from '../../res/env'
+import AsyncStorage from '@react-native-community/async-storage'
 
 const API_VERSION_STOREFRONT = '/api/v2/storefront';
 const API_ROOT = HOST;
 
-function handleAPI( path, params, method, data = null) {
+async function getAuthToken() {
+ 
```

Comment:
Remove Unwanted blank spaces and blank lines throughout the app  @vivek-mhajn

Reply:
Sure